modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -313,14 +332,16 @@ PERF_TEST_P_(DivPerfTest, TestPerformance)
     // G-API code ////////////////////////////////////////////////////////////
     cv::GMat in1, in2, out;
     out = cv::gapi::div(in1, in2, dtype);
-    cv::GComputation c(GIn(in1, in2), GOut(out));
+    cv::GComputation c(in1, in2, out);
```

Comment:
please revert this change, as unrelated to the issue

Reply:
Fixed